src/main/java/org/llorllale/cactoos/matchers/MatchesRegex.java

```diff
@@ -52,8 +53,11 @@ public MatchesRegex(final Text regex) {
         super(
             new TextMatcher(
                 new MatcherOf<>(
-                    (String act) -> act.matches(regex.asString()),
-                    regex
+                    act -> act.matches(regex.asString()),
+                    desc -> desc.appendText(
+                        new FormattedText("\"%s\"", regex).asString()
```

Comment:
@baudoliver7 for regex, let's have the regex be wrapped in `/` instead of `"` and let's use Pattern.compile().toString() to get a proper representation of the regex in string form

Reply:
@victornoel Modifications done :)